app/test_main.py

```diff
@@ -2,2 +1,46 @@
+import pytest
+from app import main
 
-# write your code here
+
+def convert_to_human(
+    animal_age: int,
+    first_year: int,
+    second_year: int,
+    each_year: int
+) -> int:
+    if animal_age < first_year:
+        return 0
+    if animal_age < first_year + second_year:
+        return 1
+    return 2 + (animal_age - first_year - second_year) // each_year
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
```

Comment:
The test cases do not cover edge cases where the function might receive data out of the normal range, such as negative numbers or extremely large numbers. According to the checklist, you should ensure that your tests check all edge situations.